devices/philips.js

```diff
@@ -1067,6 +1067,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LCW002', '4090231P9'],
```

Comment:
I don't expect both are needed, can you provide the data/database.db entry of your device?

Reply:
{"id":6,"type":"Router","ieeeAddr":"0x001788010922a605","nwkAddr":28163,"manufId":4107,"manufName":"Philips","powerSource":"Mains (single phase)","modelId":"4090231P9","epList":[11,242],"endpoints":{"11":{"profId":260,"epId":11,"devId":269,"inClusterList":[0,3,4,5,6,8,4096,64515,768,64513],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"4090231P9","manufacturerName":"Philips","powerSource":1,"zclVersion":2,"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210331","swBuildId":"1.88.1"}},"lightingColorCtrl":{"attributes":{"colorCapabilities":31,"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210331","swBuildId":"1.88.1","zclVersion":2,"interviewCompleted":true,"meta":{"configured":88764544},"lastSeen":1637341422247,"useImplicitCheckin":true}